crates/apub/src/objects/private_message.rs

```diff
@@ -107,7 +107,9 @@ impl Object for ApubPrivateMessage {
     check_apub_id_valid_with_strictness(note.id.inner(), false, context).await?;
     let person = note.attributed_to.dereference(context).await?;
     if person.banned {
-      return Err(LemmyErrorType::PersonIsBannedFromSite)?;
+      return Err(LemmyErrorType::PersonIsBannedFromSite(
+        person.id.to_string(),
```

Comment:
This is a numeric id, not really helpful. Use `person.actor_id` which is like `https://lemmy.ml/u/example` which is easy to understand.